Tools/CISupport/Shared/de-duper

```diff
@@ -96,16 +97,20 @@ def main(args):
             sha_value, extension = existing_file.name.split('.', 1)
             if extension != 'png':
                 continue
-            if not existing_file.inode() <= 1:
+            connected_to = existing_file.inode()
+            if connected_to <= 1:
                 if args.dry_run:
                     print(f'[dry-run] Removing {existing_file.path}')
                 else:
                     os.remove(existing_file.path)
                 cleaned_up += 1
             else:
-                existing_links.add(sha_value)
+                existing_links[sha_value] = connected_to
```

Comment:
The caching of the inode count makes me slightly nervous, because while we probably don't expect external processes to mutate it, there is always that risk.